meemoo/helpers.py

```diff
@@ -89,6 +89,8 @@ def get_from_event(event, name):
         return try_to_find_md5(record["s3"]["object"]["metadata"])
     elif name == "event_name":
         return record["eventName"]
+    elif name == "size":
+        return record["s3"]["object"]["size"]
```

Comment:
Dit verwacht dat het event wel degelijk deze key bevat maar de key zit niet in `REQUIRED_S3_FIELDS` lijst.

Dit is beetje een obscuur en onduidelijk systeem. Dit is de check die gebeurt: 
```
assert set(REQUIRED_S3_FIELDS).issubset(set(fields_present_in_event))
```
Als het veld ook in `REQUIRED_S3_FIELDS` zit, assert die of de waarde van die key niet "leeg" is.

Samengevat:
-  Als de key niet in het event zit, wordt er een `KeyException` gegooid wat leidt tot een `InvalidEventException`.
-  Daarna wordt getest of de waarde van de key niet "leeg" is. Als die key in de lijst `REQUIRED_S3_FIELDS` zit maar een lege waarde heeft, leidt dit ook tot een `InvalidEventException`.  

De size, dat is een `int`. De waarde 0 is een "lege" waarde in Python.  Momenteel moet die key aanwezig zijn, maar de waarde mag 0 zijn. Dit lijkt me contradictorisch.

Alhoewel het niet voor alle velden zo is, maar bij een toevoeging van een veld is het een idee om ook een negatieve test te maken.

Reply:
> Je wil een test om te controleren dat een KeyException gegooid wordt indien size ontbreekt?
Ja, dat lijkt me een goeie. Mogelijk kunnen die testen wat opgekuist worden via `pytest.mark.parametrize`.

>In de code moet er niet per se een assert komen die checkt dat size > 0?
Als we dat willen, moet die "gewoon" toegevoegd worden aan `REQUIRED_S3_FIELDS`. Either way, sowieso niet slecht om een test te hebben dat ook die logica checkt.